### ISSUE TITLE
Use of skos:definition and rdfs:comment

### ISSUE THREAD

**Issue by DieterDePaepe** [inspire-eu-rdf/inspire-rdf-guidelines#40]
[The guidelines](http://inspire-eu-rdf.github.io/inspire-rdf-guidelines/#ref_cr_general_documentation) currently state that `skos:definiton` should be used for definitions and `rdfs:comment`.

Although I find myself in this statement, I do see a conflict in the conformance to the actual world, as all ISA/W3C that I am familiar with in a government context seem to only use `rdfs:comment` as the predicate to define their terms.

- [Core location](https://www.w3.org/ns/locn.ttl)
- [Core organization](https://www.w3.org/ns/org.ttl)
- [Registered Organization](https://www.w3.org/ns/regorg.ttl)
- European Legislation Identifier
- [Core public service](http://purl.org/vocab/cpsv)
- [Core person](https://joinup.ec.europa.eu/asset/core_person/asset_release/core-person-vocabulary)

**Comment by namedgraph** [comment]
Agreed, SKOS is really out place here. `rdfs:label` should be used for TBox term labels.
